docs/issue-to-earn.mdx

```diff
@@ -0,0 +1,319 @@
+
+**What do you want to implement?**
+
+<Tabs groupId="options" defaultValue="free">
+
+<TabItem value="free" label="Free Only">
+
+  **You need to issue <Tip type="credential">credentials</Tip> to a user.** When you do, Verified Inc. will pay you according to your contract.
+
+  1. **Get user consent.** Ask the user if they want to activate their digital ID card (from your brand), and include Verified Inc. legal language:
+
+  > By clicking the button below, you agree to Verified Inc.’s <a href="https://www.verified.inc/legal/terms-of-use">Terms of Use</a> and direct YOUR_COMPANY_NAME to share your personal information with Verified Inc. to provide its digital ID card services, in accordance with its <a href="https://www.verified.inc/legal/privacy-policy">Privacy Policy</a>.
+
+  "Terms of Use" should be underlined or colored and hyperlinked to <a href="https://www.verified.inc/legal/terms-of-use">`https://www.verified.inc/legal/terms-of-use`</a>. "Privacy Policy" should be underlined or colored and hyperlinked to <a href="https://www.verified.inc/legal/privacy-policy">`https://www.verified.inc/legal/privacy-policy`</a>.
+
+  :::info
+  **You must obtain user consent before issuing credentials.** Not doing so violates our terms and will prevent us from paying you for issuing credentials. 
+  :::
+
+  2. **Call [`/credentials`](/api-overview#issue-credentials)** with the user's verified email and/or phone and one or more credentials. For each credential, include a `type` and `data` in accordance with one of our [schemas](/schema). 
+
+  _If we don't have a schema that suits your needs, [email us](mailto:support@verified.inc) and we'll make one for you._
+
+  For example, if you have a verified Social Security Number (SSN) for a user, you can issue them a `SsnCredential`. Just call the `/credentials` endpoint with the user's email and/or phone, the `SsnCredential` type, and the SSN data:
+
+  ```json title="Example Issue Credentials Request Body"
+  {
+    "email": "test@example.com",
+    "phone": "+10123456789",
+    "credentials": [
+      {
+        "type": "SsnCredential",
+        // based on the properties attribute of the schema
+        // highlight-start
+        "data": {
+          "ssn": "111223333"
+        },
+        // highlight-end
+        "expirationDate": 1893484800
+      }
+    ]
+  }
+  ```
+
+  Format the credential data according to the `properties` attribute of the credential schema. In this case, the schema is for a `SsnCredential` which can found found at [`https://schema.verified.inc/jsonSchema/SsnCredential`](https://schema.verified.inc/jsonSchema/SsnCredential). 
+
+  More information about credential schemas can be found in the [Schema](/schema) section.
+
+  ✅ **_ You're done!_**
+
+  > For more detailed information on how and why to implement Free IDV, see our [Free IDV Guide](/issuance-guide).
+
+</TabItem>
+  
+<TabItem value="1-click" label="1-Click Only">
```

Comment:
I feel like it could be petty confusing as to why we are not using the /1-click endpoint here. Why aren't we? Couldn't it be used in this flow, too (as long as they use the phone as a user identifier)?

Reply:
I haven't edited this page yet. It's just retitled but otherwise it's exactly the same as the old issuance guide